thirdeye/thirdeye-frontend/tests/acceptance/edit-alert-test.js

```diff
@@ -3,27 +3,23 @@ import moduleForAcceptance from 'thirdeye-frontend/tests/helpers/module-for-acce
 
 moduleForAcceptance('Acceptance | edit alert');
 
-const EDIT_BUTTON = '.te-search-results__edit-button a';
 const METRIC_NAME = `#select-metric`;
 const GRANULARITY = `#select-granularity`;
 const ALERT_NAME = `#anomaly-form-function-name`;
 const STATUS = '.te-toggle--form span';
+const EDIT_LINK = '/manage/alerts/edit';
+const STATUS_TOGGLER = '.x-toggle-btn';
+const SUBMIT_BUTTON = '.te-button--submit';
 
-test('visiting /manage/alerts/edit and checking that fields render correctly', assert => {
-  visit('/manage/alerts');
+// TODO: Update the syntax to use async/await http://rwjblue.com/2017/10/30/async-await-configuration-adventure/
+test(`visiting ${EDIT_LINK} and checking that fields render correctly and edit is successful`, assert => {
+  let alert = server.create('alert');
```

Comment:
are we expected to change the `alert` here? should we use a `const`?

Reply:
You're right. Will fix this.